pkg/rc/farm.go

```diff
@@ -89,7 +88,7 @@ func NewFarm(
 // Set the metrics registry to use (see
 // https://godoc.org/github.com/rcrowley/go-metrics#Registry). If set,
 // performance metrics will be recorded to the registry.
-func (rcf *Farm) SetMetricsRegistry(registry metrics.Registry) error {
+func (rcf *Farm) SetMetricsRegistry(registry rcmetrics.MetricsRegistry) error {
```

Comment:
Is it that you think people might want to do things with rc_processing_time other than put them in a Histogram?